api/apps/graphql/resolvers.coffee

```diff
@@ -32,4 +33,9 @@ module.exports.channels = (root, args, req, ast) ->
 module.exports.tags = (root, args, req, ast) ->
   return new Promise (resolve, reject) ->
     Tag.where args, (err, results) ->
+      resolve results.results
+
+module.exports.authors = (root, args, req, ast) ->
+  return new Promise (resolve, reject) ->
```

Comment:
ast stands for abstract syntax tree. you can think of it like an internal tree/json/schema that Graphql uses internally to process your request. your query gets converted to an ast - a format that GraphQL understands. try printing it out next time!